manage_user_create.php

```diff
@@ -68,9 +68,9 @@
 
 # Password won't to be sent by email. It entered by the admin
```

Comment:
I would suggest to take the opportunity to rephrase this in proper English
`Password won't be sent by email. It is entered by the admin`